background/README.md

```diff
@@ -0,0 +1,2 @@
+This directory contains documents that aren't part of any particular language
+problem or solution but contain useful reference information for us.

```

Comment:
us -> discussions about language features.

(Otherwise, who is "us"?)

Reply:
Done.